src/Main.elm

```diff
@@ -1460,11 +1479,11 @@ decodeFormField =
 encodeInputField : InputField -> Json.Encode.Value
 encodeInputField inputField =
     case inputField of
-        ShortText inputType maybeMaxLength ->
+        ShortText inputType attrs ->
             Json.Encode.object
                 [ ( "type", Json.Encode.string "ShortText" )
                 , ( "inputType", Json.Encode.string inputType )
-                , ( "maxLength", maybeMaxLength |> Maybe.map Json.Encode.int |> Maybe.withDefault Json.Encode.null )
+                , ( "attributes", attrs |> List.map (Tuple.mapSecond Json.Encode.string) |> Json.Encode.object )
```

Comment:
prefer we don't change the JSON to nest under a new `"attributes"` key?

we should instead

```elm
(attrs ++ [( "type", "ShortText"), ("inputType", inputType)])
    |> Dict.fromList
    |> Json.Encode.dict identity Json.Encode.string
```

- https://package.elm-lang.org/packages/elm/core/latest/Dict#fromList
- https://package.elm-lang.org/packages/elm/json/latest/Json-Encode#dict

Reply:
actually nesting is necessary since we're already using `"type"` to encode `InputField` variants 🫣